libraries/stdlib/genmdl/stdlib.mdl

```diff
@@ -0,0 +1,7751 @@
+mdl 1.6;
```

Comment:
I'm not sure this file should be merged in. As discussed on slack Nvidia will create an MDL package, including both stdlib, pbrlib and supplemental nodes, and this will be based on the implementations from Allegorithmic.

Reply:
This I can leave out as it's just a stub file.